documents/component-guide.md

```diff
@@ -0,0 +1,242 @@
+## Component Guide
+
+The main function of this document is to introduce some of the main `containers` & `elements` and their usages that we have in this project.
+
+### Section Container
```

Comment:
Should we use to component name in the header, so it is easier to look for.
`SectionContainer` instead of `Section Container`